docs/modules/integrations/pages/applitools-eyes/applitools-eyes-integration.adoc

```diff
@@ -0,0 +1,58 @@
+= Enable Applitools Eyes integration

```

Comment:
Remove "Enable" from title. Looking at all the other integration pages, this isn't part of the title